UniversalAlgebra/Relations/Quotients.lagda

```diff
@@ -49,6 +49,27 @@ Antisymm _≈_ = ∀{x}{y} → x ≈ y → y ≈ x → x ≡ y
 Trans : {A : Type 𝓤} → Rel A 𝓦 → Type(𝓤 ⊔ 𝓦)
 Trans _≈_ = ∀{x}{y}{z} → x ≈ y → y ≈ z → x ≈ z
 
+Equivalence : {𝓤 : Level} → Type 𝓤 → Type (lsuc 𝓤)
+Equivalence {𝓤} A = Σ[ r ꞉ Rel A 𝓤 ] IsEquivalence r
+
+\end{code}
+
+
+\begin{code}
+
+module _ {I : Type 𝓥} {A : Type 𝓤 } where
+
+ 𝟎 : Rel (I → A) (𝓥 ⊔ 𝓤)
+ 𝟎 x y = ∀ i → x i ≡ y i
+
+
+ 𝟎-IsEquivalence : IsEquivalence 𝟎
+ 𝟎-IsEquivalence = record { refl = λ i → refl ; sym = λ α i → sym (α i)  ; trans = λ α β ℓ → trans (α ℓ) (β ℓ) }
+
+𝟎-is-smallest : Setω
```

Comment:
Do you need the theorem to hold globally, i.e. that for all levels `𝟎-is-smallest`? Usually, it is sufficient to have that at any given collection of levels, `𝟎-is-smallest`, i.e. move the quantification over the 3 levels out, and then the result itself will be at a finite level too.